config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml

```diff
@@ -15,4 +15,43 @@ periodics:
     containers:
     - image: gcr.io/k8s-testimages/kubekins-e2e:v20190220-615bdbe2e-1.12
       command:
-      - "./scripts/ci-aws-cred-test.sh"
\ No newline at end of file
+      - "./scripts/ci-aws-cred-test.sh"
+- name: periodic-cluster-api-provider-aws-bazel-e2e
+  decorate: true
+  interval: 1h
+  labels:
+    preset-service-account: "true"
+    preset-aws-ssh: "true"
+    preset-aws-credential: "true"
+  extxa_refs:
+  - org: kubernetes-sigs
+    repo: cluster-api-provider-aws
+    base_ref: master
+    path_alias: "sigs.k8s.io/cluster-api-provider-aws"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190220-615bdbe2e-1.12
+      command:
+      - make
+      args:
+      - e2e
+      env:
+      - name: BOSKOS_HOST
+        value: "http://boskos.test-pods.svc.cluster.local."
+postsubmits:
+  kubernetes-sigs/cluster-api-provider-aws:
+  - name: pull-cluster-api-provider-aws-bazel-e2e
+    decorate: true
+    max_concurrency: 5
```

Comment:
Hi @detiber,

FYI, there will be at most `5` concurrent builds off of `master`. This gives back five shared accounts, one of which can then be used for the periodic.

Reply:
Do we have enough accounts available for this? I wasn't tracking how many accounts we had total for this use case.

Reply:
Hi Jason,

There are 10 shared accounts. It’s never been clear to me how many are available to CAPA as they’re all named CAPA something. To an outsider like me it would seem *all* the accounts are available.